plugins/performance-lab/includes/site-health/cache-control-headers/helper.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Helper functions used for Cache-Control headers site health check.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Tests the Cache-Control headers.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function perflab_cch_check_cache_control_test(): array {
+	$result = array(
+		'label'       => __( 'Cache settings are optimal for site performance.', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => '<p>' . esc_html__( 'Your site cache settings are configured correctly, helping to improve its performance.', 'performance-lab' ) . '</p>',
+		'actions'     => '',
+		'test'        => 'perflab_cch_cache_control_header_check',
+	);
+
+	$response = wp_remote_get(
+		home_url(),
```

Comment:
To avoid potential redirect:

```suggestion
		home_url( '/' ),
```